unitypack/type.py

```diff
@@ -79,7 +80,9 @@ def load_blob(self, buf):
 			curr.size = buf.read_int()
 			curr.index = buf.read_uint()
 			curr.flags = buf.read_int()
-			unk1 = buf.read_int64() # zero bytes?
+
+			if self.format >= 20:
```

Comment:
I would leave a comment indicating the uncertainty of versions 18 and 19 here, unless you solve it more definitively.